agent/src/agent/pipeline/pipeline.py

```diff
@@ -181,7 +181,15 @@ def dimension_configurations(self) -> Optional[dict]:
                 'Pipeline dimensions should be a list in order to build dimension_configurations, '
                 f'but {type(self.dimensions).__name__} provided'
             ))
-        return _build_dimension_configurations(self.dimensions, self.config.get('dimension_configurations'))
+        return _build_transformation_configurations(self.dimensions, self.config.get('dimension_configurations'))
+
+    @property
+    def values_configurations(self) -> Optional[dict]:
+        return _build_transformation_configurations(list(self.values), self.config.get('values_configurations', {}))
+
+    @property
+    def tags_configurations(self) -> Optional[dict]:
+        return self.config.get('tags_configurations', {})
```

Comment:
why don't we have build here?

Reply:
To allow such tags combination in a config file
```
"tags": 
    {
      "some_static_tag": ["tag_values"]
    },
    "tag_configurations": {
      "INTERFACE_ALIAS": {
        "value_path": "ifAlias"
      },
    }
```

If the build function will be used, then its output will be like this:
```
{'INTERFACE_ALIAS': {'value_path': 'ifAlias'},
 'some_static_tag': {'value_path': 'some_static_tag'}}  # and we don't have  this value_path': 'some_static_tag'
```